src/com/osudroid/ui/v2/BeatmapInfoLayout.kt

```diff
@@ -0,0 +1,260 @@
+
+        flexContainer {
+            width = FillParent
+
+            linearContainer {
+                flexRules {
+                    grow = 1f
+                }
+                orientation = Orientation.Vertical
+
+                artistText = text {
+                    font = ResourceManager.getInstance().getFont("smallFont")
+                    text = "Unknown"
+                    applyTheme = { color = it.accentColor * 0.9f}
+                }
+
+                titleText = text {
+                    width = FillParent
+                    font = ResourceManager.getInstance().getFont("smallFont")
+                    text = "No selected beatmap"
+                    applyTheme = { color = it.accentColor }
+                    clipToBounds = true
+                }
+
+                versionText = text {
+                    font = ResourceManager.getInstance().getFont("smallFont")
+                    text = "Unknown"
+                    applyTheme = { color = it.accentColor * 0.8f}
+                }
+            }
+
+            starsText = badge {
+                text = "0.00"
+                leadingIcon = UISprite(ResourceManager.getInstance().getTexture("star-xs"))
+            }
+        }
+
+        linearContainer {
+            orientation = Orientation.Horizontal
+            spacing = 8f
+
+            lengthText = compoundText {
+                leadingIcon = UISprite(ResourceManager.getInstance().getTexture("clock"))
+                text = "00:00"
+                applyTheme = { color = it.accentColor }
+                font = ResourceManager.getInstance().getFont("xs")
+            }
+
+            bpmText = compoundText {
+                leadingIcon = UISprite(ResourceManager.getInstance().getTexture("bpm"))
+                text = "0"
+                applyTheme = { color = it.accentColor }
+                font = ResourceManager.getInstance().getFont("xs")
+            }
+        }
+
+        linearContainer {
+            orientation = Orientation.Vertical
+            spacing = 4f
+
+            linearContainer {
+                orientation = Orientation.Horizontal
+                spacing = 4f
+
+                circlesBadge = labeledBadge {
+                    label = "Circles"
+                    value = "0"
+                    sizeVariant = SizeVariant.Small
+                }
+                slidersBadge = labeledBadge {
+                    label = "Sliders"
+                    value = "0"
+                    sizeVariant = SizeVariant.Small
+                }
+                spinnersBadge = labeledBadge {
+                    label = "Spinners"
+                    value = "0"
+                    sizeVariant = SizeVariant.Small
+                }
+            }
+
+            linearContainer {
+                orientation = Orientation.Horizontal
+                spacing = 2f
+
+                arText = labeledBadge {
+                    label = "AR"
+                    value = "00.0"
```

Comment:
These should default to `0.00` and not `00.0` (I don't recall any other place in the game that uses this format), even the star rating above uses `0.00`.